go-controller/pkg/node/ovspinning/ovspinning_linux.go

```diff
@@ -65,6 +69,20 @@ func Run(ctx context.Context, stopCh <-chan struct{}, podResCli podresourcesapi.
 		defer fileWatcher.Close()
 	}
 
+	// we only need to check reservedSystemCPUs once at startup.
```

Comment:
but its not run always on startup right? its only if the dynamic ovs cpu pinning feature is on i guess?

Reply:
Right. If the feature is off, the`Run()` call returns early and doesn't perform this check.

Reply:
no I meant when the feature is on... is it done only once at startup? - hopefully yes and not during every periodic check